### ISSUE TITLE
Network of two nodes?

### ISSUE THREAD

**Issue by jhammock** [danielabar/globi-proto#30]
Hello!

I love the explorer -> network connection but I'm not quite sure what happens in some cases when I use it.

Starting at http://danielabar.github.io/globi-proto/#/main?name=Thunnus%20albacares&interaction=hasParasite  and selecting the first parasite, Nasicola klawei, I get no network results. 

I have a theory, that the default network shown takes the interaction type in the explore window and propagates it to any organisms "receiving" the relationship received by the network focus (Nasicola klawei) from the original explore focus (Yellowfin tuna). That works pretty well for eats or preys on, but not for host-parasite, host-pathogen, vector-pathogen, symbiont, interacts, or pollinator-plant. possibly other visualizations would be more useful for these cases. The bipartite graph would be good for the ones where there are two clear pools of partners, but no further layers; like host-pathogen, plant-pollinator, etc. 

I still think in layered cases, like eats or preys on, the network display should be backed up one level, since there is always an "eats" interaction involving the focus taxon available to display. For instance, selecting Manatee grass from http://danielabar.github.io/globi-proto/#/main?name=Chelonia%20mydas&interaction=eats could start at the Green turtle and show all the "sister food items", since there's nothing to show in the "Manatee grass eats X" direction.

**Comment by danielabar** [comment]
Yes the way it works currently is confusing. This will be addressed by #44 so that there is only one way to get into the Network view, given that a subject taxon has been selected.

The other piece that will  help is #35 , which allows for the starting point of the network to be modified based on selected interaction, within the network tab. Actually this second part is already deployed and can be tried out live.

Thank you so much for your feedback!

**Comment by jhammock** [comment]
Just went back and wandered through four levels from yellowfin tuna. This is lovely!

**Comment by danielabar** [comment]
Thanks that's great to hear :-)
